pkgs/development/compilers/ghc/8.0.1.nix

```diff
@@ -57,6 +57,8 @@ stdenv.mkDerivation rec {
   stripDebugFlags = [ "-S" ] ++ stdenv.lib.optional (!stdenv.isDarwin) "--keep-file-symbols";
 
   postInstall = ''
+    paxmark m $out/lib/${name}/bin/{ghc,haddock}
```

Comment:
What will this command do when it's run on a platform other than Linux? Do Macs have `paxmark`? Do all variants of  Linux have it and is it always in `$PATH` like it's assumed here?